engines/scumm/scumm.cpp

```diff
@@ -2315,14 +2315,6 @@ void ScummEngine::scummLoop(int delta) {
 		}
 	}
 
-	// Trigger autosave if necessary.
-	if (!_saveLoadFlag && shouldPerformAutoSave(_lastSaveTime) && canSaveGameStateCurrently()) {
```

Comment:
I do not see the check for `canSaveGameStateCurrently()` in the global autosave code. Perhaps, it should be added there?

Reply:
That's in engine.h:

```
	virtual bool canSaveAutosaveCurrently() {
		return canSaveGameStateCurrently();
	}
```